.gitignore

```diff
@@ -23,7 +23,9 @@ CMakeLists.txt.user
 # Project specific 'ignore patterns'
 #
 # Put you project specific ignores below.
-# If they are generally useful consider adding them to the 
```

Comment:
This line must be identical to the corresponding line in the project-template...